tests/robot-tests/tests/public_api/public_api_minor_manual_changes.robot

```diff
@@ -0,0 +1,386 @@
+    user waits until h3 is visible    Filter options not found in new data set
+    user checks table column heading contains    1    1    Current data set
+    user checks table column heading contains    1    2    New data set
+
+    user checks table column heading contains    1    3    Type
+    user checks table column heading contains    1    4    Actions
+
+    user checks table cell contains    1    1    Total
+    user checks table cell contains    1    2    State-funded primary and secondary
+    user checks table cell contains    1    3    Minor
+
+    user clicks link    Back
+
+Confirm finalization of this API data set version
+    user clicks button    Finalise this data set version
+    user waits for caches to expire
+    user waits until h2 is visible    Mappings finalised
+    user waits until page contains    Draft API data set version is ready to be published
+
+User navigates to 'changelog and guidance notes' page and update relevant details in it
+    user clicks link by index    View changelog and guidance notes    1
+    user waits until page contains    API data set changelog
+
+    user enters text into element    name:notes
+    ...    Content for the public guidance notes
+    user clicks button    Save public guidance notes
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+User clicks on 'View preview token log' link inside the 'Draft version details' section
+    user clicks link by index    View changelog and guidance notes    2
+
+Validate the contents in the 'API data set changelog' page.
+    user waits until page contains    API data set changelog
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+Add headline text block to Content page for the second release
+    user navigates to content page    ${PUBLICATION_NAME}
+    user adds headlines text block
+    user adds content to headlines text block    Headline text block text
+
+Approve second release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+Verify newly published release is on Find Statistics page
+    user checks publication is on find statistics page    ${PUBLICATION_NAME}
+
+User navigates to data catalogue page
+    user navigates to data catalogue page on public frontend
+
+Search for the new API data set version
+    user clicks element    name:search
+    user presses keys    ${PUBLICATION_NAME}
+    user clicks radio    API data sets only
+
+    user waits until page finishes loading
+    user clicks radio    Newest
+
+    user checks summary list contains    Status    This is the latest data
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
+    user checks summary list contains    Status    Available by API
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
```

Comment:
Similar to other comments - don't need the named parameter?

Reply:
Done